src/rules/indent-helpers/svelte.ts

```diff
@@ -38,7 +43,7 @@ export function defineVisitor(context: IndentContext): NodeListener {
             node.children.filter(isNotEmptyTextNode),
             node.startTag,
             node.endTag,
-            1,
+            options.indentScript ? 1 : 0,
```

Comment:
I don't think this change is necessary. Did we need this change?

Reply:
Unclear, I'll test without tomorrow and see what happens.